### ISSUE TITLE
WCAG 1.3.1: Ensures <li> elements are used semantically (li:nth-child(8))

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#7]
#### Issue
`Ensures <li> elements are used semantically` ([listitem](https://dequeuniversity.com/rules/axe/3.5/listitem?application=msftAI))
#### Target application
[EO Redirect](http://localhost:8080/public/jsp/Utilities/EO_Redirect.myjsp)
#### Element path
li:nth-child(8)
#### Snippet
`<li>
		<a class="pageSubNavTxt" href="/public/jsp/Utilities/EO_13497.pdf">Executive Order 13497, Revocation of Certain Executive Orders Concerning Regulatory Planning and Review</a> (74 FR 6113; February 4, 2009)
		</li>`
#### How to fix
    Fix any of the following:
      List item does not have a <ul>, <ol> parent element
#### Environment
Chrome version 81.0.4044.62
This accessibility issue was found using Accessibility Insights for Web 2.15.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
